src/scenes/Species/SpeciesDetailView.tsx

```diff
@@ -178,7 +178,7 @@ export default function SpeciesDetailView({ reloadData }: SpeciesDetailViewProps
             <TextField
               id={'growthForm'}
               label={strings.GROWTH_FORM}
-              value={featureFlagMockedSpecies ? species?.growthFormNEXT?.join(', ') : species?.growthForm}
+              value={featureFlagMockedSpecies ? species?.growthFormNEXT?.join(', ') : species?.growthForms?.join(', ')}
```

Comment:
This can now be `species?.growthForms?.join(', ')`, the `featureFlagMockedSpecies` is not necessary